blockchain/blockchain.go

```diff
@@ -121,16 +120,33 @@ func SimulateExecution(bc Blockchain, caller address.Address, ex *action.Executi
 	if err != nil {
 		return nil, nil, err
 	}
-	raCtx := protocol.MustGetRunActionsCtx(ctx)
-	raCtx.Caller = caller
-
+	bcCtx := protocol.MustGetBlockchainCtx(ctx)
+	ctx = protocol.WithActionCtx(
+		ctx,
+		protocol.ActionCtx{
+			Caller: caller,
+		},
+	)
+	zeroAddr, err := address.FromString(address.ZeroAddress)
+	if err != nil {
+		return nil, nil, err
+	}
+	ctx = protocol.WithBlockCtx(
+		ctx,
+		protocol.BlockCtx{
+			BlockHeight:    bcCtx.Tip.Height,
+			BlockTimeStamp: bcCtx.Tip.Timestamp,
```

Comment:
bcCtx.Tip.Timestamp + bcCtx.Genesis.BlockInterval
or
time.Time{}

Reply:
fixed in latest commit